mantis-connectors/mantis-connector-iceberg/src/main/java/io/mantisrx/connector/iceberg/sink/writer/config/WriterConfig.java

```diff
@@ -34,6 +34,8 @@ public class WriterConfig extends SinkConfig {
     private final int writerMaximumPoolSize;
     private final Configuration hadoopConfig;
 
+    private final boolean watermarkEnabled;
```

Comment:
wondering if this property can be removed entirely. Writers always calculate the low watermark and always populate to committer stage. User can simply provide null value (default) to disable this.

Reply:
I think we can depend on the watermarkPropertyKey being null or not to check if this is enabled? So that user will not need to input this config value but it's kinda derived.